Frontend/src/test/pages/dashboard/components/Customer/AddressCard.test.jsx

```
@@ -0,0 +1,103 @@
+import { AddressCard } from '../../../../../pages/dashboard/components/Customer/AddressCard'
+
+describe('<AddressCard />', () => {
+  const sample = {
+    name:    'Home',
+    street:  '123 Main St',
+    city:    'Testville',
+    state:   'TS',
+    zip:     '12345',
+    country: 'Testland',
+  }
+  let onEdit, onDelete
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)